typeshed_client/parser.py

```diff
@@ -356,30 +353,22 @@ class _LiteralEvalVisitor(ast.NodeVisitor):
     def __init__(self, ctx: SearchContext) -> None:
         self.ctx = ctx
 
-    # from version 3.8 on all constants are represented as ast.Constant
-    if sys.version_info < (3, 8):
-
-        def visit_Num(self, node: ast.Num) -> Union[int, float, complex]:
-            return node.n
-
-        def visit_Str(self, node: ast.Str) -> str:
-            return node.s
-
-    else:
+    def visit_Num(self, node: ast.Num) -> Union[int, float, complex]:
+        return node.n
 
-        def visit_Constant(self, node: ast.Constant) -> Any:
-            return node.value
+    def visit_Str(self, node: ast.Str) -> str:
+        return node.s
```

Comment:
I think you deleted the wrong branch here -- you want to keep the `visit_Constant` method but delete the `visit_Num` and `visit_Str` methods

Reply:
Good point, thanks